operations/governance-remediation-process.md

```diff
@@ -0,0 +1,83 @@
+
+## Pre-escalation expectations
+
+Project members and stakeholders should first attempt to resolve governance issues through normal project discussion and consensus. The Remediation process exists for when normal, in-project resolution fails, either because of a difference of opinion, or because the project lacks the tools to make sufficient changes themselves.
+
+## Governance Remediation
+
+### Objectives:
+
+#### Establish Missing Governance 
+
+_Assist projects in developing governance structures to handle situations they are currently unequipped to manage._
+
+Missing governance can usually be remediated within the project by agreeing to adopt the missing governance. This is commonly amicable and not very contentious within a project as the cause or driving mechanism results from a lack of a clear path forward. In this case, missing governance is an opportunity for improvement. 
+
+#### Correct Non-Practiced Governance 
+
+_Ensure projects adhere to their documented governance._
+
+Non-practiced governance can either be someone willfully or inadvertently ignoring stated governance. Alternatively, occasions may arise where the defined governance _cannot_ be practiced as the project may be lacking people in a role to execute governance processes.
+
+In the case of inadvertent ignorance, these may be resolved internally to the project by pointing out the existing governance against which the behavior or processes must be brought into alignment. 
+
+In circumstances where the project lacks the people in the defined roles to execute governance processes, the TOC with support from TAG Contributor Strategy serves as an independent advisor to recommend and guide the project to a fair, transparent outcome. In some cases, making specific recommendations to the project to resolve the issue.  If no other course is available, the TOC will directly approve changes to project governance in order to allow it to operate again.
+
+In cases where leaders of an open source project willfully disregard the project's published governance or CNCF principles around open governance, the TOC, with advice from the Governance WG, will prepare a remediation plan. This plan could involve supervised project administration, changes to the written governance, removal of specific project leaders, or whatever other measures are required to restore the project to a state where it is following its own governance rules, and that governance is in accord with CNCF principles. Should there be no way to restore the project to community operation, the TOC will consider archiving the project.
```

Comment:
It strikes me as a concern that the only lever available to the TOC is to archive a project.

Reply:
Do we have examples of maintainers wilfully disregarding their governance, where that wouldn't already be covered by the Code of Conduct?